velox/row/tests/UnsafeRowTest.cpp

```diff
@@ -235,5 +293,23 @@ TEST_F(UnsafeRowFuzzTests, fast) {
       });
 }
 
+TEST_F(UnsafeRowTests, nestedMaps) {
+  auto innerMaps = makeRowVector(
+      {makeNullableArrayVector<int64_t>({
+           {{1, 2, std::nullopt, 3}},
+           {{4, 5}},
+           {{1}},
+           std::nullopt,
+           {{3, 2, 4, 5}},
+           {{6}},
+       }),
+       makeNullableFlatVector<int32_t>({1, 2, 3, std::nullopt, 5, 6})},
+      [](auto row) { return row == 3; });
+  auto keys = makeFlatVector<int32_t>({1, 2, 3, 4, 5, 6});
+  auto values = makeMapVector({0, 2, 2, 3, 4, 5}, keys, innerMaps, {1});
```

Comment:
Wondering if using makeMapVectorFromJson would make the test more readable.

Reply:
No, makeMapVectorFromJson can only generate the map vector with primitive key and value data type. The value type is ROW in this test.